ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js

```diff
@@ -75,7 +75,11 @@ export default function SecureYourWallet() {
         </Typography>
       </Box>
       <Box>
-        <video controls style={{ borderRadius: '10px' }}>
+        <video
+          className="secure-your-wallet__video"
+          controls
+          style={{ borderRadius: '10px' }}
```

Comment:
Any reason this style is inline rather than in the stylesheet?

Reply:
Not sure, that was already there. I just added the className.

Reply:
Sorry @dragana8 just seeing this again... Would you mind moving it to the stylesheet?

Reply:
@adonesky1 done.